README.md

```diff
@@ -7,8 +7,7 @@ and usually involves acquiring a lock, so it becomes a kind of
 synchronisation point.
 
 The Lightweight Logging Library ([l3](./include/l3.h)) has a fast but limited `printf` API:
-- `l3_log_simple()`: Takes about 10ns (Intel Core i7-1365U)
-- `l3_log_fast()`: An even faster, but more limited interface, which takes about 7ns.
+- `l3_log()`: Takes about 2ns (Intel Core i7-1365U)
```

Comment:
? Is the updated comment about `2ns` accurate?

In old comment, C-fn was cited to take 10ns and assembly to take ~7ns.

Where did you get this 2ns from?

Kindly investigate and correct this number. Otherwise, future readers will be mislead, no ?

Reply:
Yes, this is correct. An earlier PR improved the performance. At least for single-threaded programs. I'll expand the text to be clearer.